native/service/cross_chain_manager/erd/bech32PubkeyConverter.go

```diff
@@ -0,0 +1,96 @@
+package erd
+
+import (
+	"fmt"
+
+	"github.com/btcsuite/btcutil/bech32"
+)
+
+type config struct {
+	prefix   string
+	fromBits byte
+	toBits   byte
+	pad      bool
+}
+
+var bech32Config = config{
+	prefix:   "erd",
+	fromBits: byte(8),
+	toBits:   byte(5),
+	pad:      true,
+}
+
+// bech32PubkeyConverter encodes or decodes provided public key as/from bech32 format
+type bech32PubkeyConverter struct {
+	len int
+}
+
+// NewBech32PubkeyConverter returns a bech32PubkeyConverter instance
+func NewBech32PubkeyConverter(addressLen int) (*bech32PubkeyConverter, error) {
```

Comment:
do we need this ? we have the same implementation in elrond-go. or in elrond-core

Reply:
implementation is in elrond-go-core and expects elrond logger
func NewBech32PubkeyConverter(addressLen int, log core.Logger) (*bech32PubkeyConverter, error)